varcode/variant_annotator.py

```diff
@@ -11,7 +12,7 @@ class VariantAnnotator(object):
     def __init__(self, ensembl_release):
         self.ensembl = pyensembl.EnsemblRelease(ensembl_release)
 
-    def describe_variant(self, variant):
+    def describe_variant(self, variant, raise_on_error=True):
```

Comment:
The constructor seems a little clumsier to me since the typical usecase seems to be:

1) Construct `variant_collection = VariantCollection(some_vcf_path)`
2) Call `variant_collection.variant_effects()`
3) Encounter an error
4) Decide the error is irrelevant
5) Call `variant_collection.variant_effects(raise_on_error=False)` to ignore the error

Having to create a new `VariantCollection` seems like a very minor hassle but it also doesn't have a clear payoff.